### ISSUE TITLE
To-do list

### ISSUE THREAD

**Issue by sks316** [sks316/JSRLoader#2]
- [x] Change references to "sks316" to "PrincessLillie"
- [x] See if anything broke lol
- [x] Speak to the dev of [Mcdonoughd/jetsetradiofuture.live](https://github.com/Mcdonoughd/jetsetradiofuture.live)
- [x] Reroute old stations to clone server
- [x] Add new stations

**Comment by sks316** [comment]
New stations added since last update:

Outer Space
https://jetsetradio.live/radio/stations/outerspace/~list.js

The Garage
https://jetsetradio.live/radio/stations/garage/~list.js

Ganjah
https://jetsetradio.live/radio/stations/ganjah/~list.js

Chiptunes
https://jetsetradio.live/radio/stations/chiptunes/~list.js

Retro Remix
https://jetsetradio.live/radio/stations/retroremix/~list.js

Classical Remix
https://jetsetradio.live/radio/stations/classical/~list.js

BonafideBloom
https://jetsetradio.live/radio/stations/bonafidebloom/~list.js

VeraFX
https://jetsetradio.live/radio/stations/verafx/~list.js

Snow-fi (seasonal, activates in December)
https://jetsetradio.live/radio/stations/snowfi/~list.js

**Comment by sks316** [comment]
Stations removed since last update:

Memories of Tokyo-To
https://jetsetradio.live/radio/stations/memoriesoftokyoto/~list.js

King for Another Day
https://jetsetradio.live/radio/stations/kingforanotherday/~list.js

Forever Is A Pretty Long Time
https://jetsetradio.live/radio/stations/elaquent/~list.js

JET MASH RADIO
https://jetsetradio.live/radio/stations/jetmashradio/~list.js

FUTURE GENERATION
https://jetsetradio.live/radio/stations/futuregeneration/~list.js

All removed content can be found at other sources, but can no longer be downloaded from JetSetRadio.live. As such, they will be removed from the next release.

You can find the removed content at the following sources:

Memories of Tokyo-To: [2 Mello's Bandcamp page](https://2mellomakes.bandcamp.com/album/memories-of-tokyo-to)

King for Another Day: [SiIvaGunner's Bandcamp page](https://gilvasunner.bandcamp.com/album/final-bout-siivagunner-king-for-another-day-tournament-original-soundtrack-vol-3) (There are two other parts of this soundtrack. [Click here](https://gilvasunner.bandcamp.com/album/first-strike-siivagunner-king-for-another-day-tournament-original-soundtrack-vol-1) to see Part 1. [Click here](https://gilvasunner.bandcamp.com/album/second-wind-siivagunner-king-for-another-day-tournament-original-soundtrack-vol-2) to see Part 2.)

Forever Is A Pretty Long Time: [Elaquent's Bandcamp page](https://elaquent-mello-catalog.bandcamp.com/album/forever-is-a-pretty-long-time)

JET MASH RADIO: [Listen on VKontakte](https://vk.com/music/playlist/196110197_83949520), sadly no official download seems to exist

FUTURE GENERATION: [SASHKO NAGANUMA's Bandcamp page](https://sashkonaganuma.bandcamp.com/album/music-for-jet-set-radio-future-generation)

**Comment by sks316** [comment]
I just discovered a JSRL "clone repo" at [Mcdonoughd/jetsetradiofuture.live](https://github.com/Mcdonoughd/jetsetradiofuture.live). It even has a website up at https://jetsetradiofuture.live, and while a bit buggy, it works (despite being pulled from the Wayback Machine) and has much of the removed content. If I can make this work, I won't have to remove these stations from JSRLoader after all and can simply patch the URLs to download from this clone instead of from JSRL itself. I'll speak with the developer of this clone and see if this is okay.
I also happened across [SleepyLark/Jet-Lag-Radio](https://github.com/SleepyLark/Jet-Lag-Radio), which seems to be an excellent resource for JSRL things.
